### DISCUSSION TITLE
Physical Disk Support

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#1888] [Ideas]
Would it be possible to add support for using Physical Disks directly? Like Flash Drives, SD Cards, or old PATA, IDE, and SCSI disks?

**Comment by Dizzy611** [comment]
Seconding this as a good idea. It's potentially dangerous, sure, but it's also a great way to potentially more easily set up disks to be used in actual retro systems

**Comment by Olddantrucker** [comment]
Some games like NFS4 High Stakes REQUIRE a physical disc or just like now you get a 'Please Insert Correct CD Rom' error!!!! 😲😲😲😲
